src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -37,44 +38,38 @@
     }
 
-    @GetMapping(value = "/networks/{networkUuid}/modifications", produces = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "Get modifications list of the default group for a network")
-    @ApiResponse(responseCode = "200", description = "List of modifications of the default group for the network")
-    public ResponseEntity<Flux<ModificationInfos>> getModifications(@Parameter(description = "Group UUID") @PathVariable("networkUuid") UUID groupUuid) {
+    @GetMapping(value = "/groups/{groupUuid}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get modifications list of a group for a network")
+    @ApiResponse(responseCode = "200", description = "List of modifications of the group for the network")
+    public ResponseEntity<Flux<ModificationInfos>> getModifications(@Parameter(description = "Group UUID") @PathVariable("groupUuid") UUID groupUuid) {
         return ResponseEntity.ok().body(networkModificationService.getModifications(groupUuid));
     }
 
-    @GetMapping(value = "/networks/{networkUuid}/elementarymodifications/{modificationUuid}", produces = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(summary = "Get an elementary modification of the default group for a network")
-    @ApiResponse(responseCode = "200", description = "Elementary modification of the default group for the network")
-    public ResponseEntity<Mono<ElementaryModificationInfos>> getElementaryModifications(@Parameter(description = "Group UUID") @PathVariable("networkUuid") UUID groupUuid,
-                                                                                        @Parameter(description = "Modification UUID") @PathVariable("modificationUuid") UUID modificationUuid) {
-        return ResponseEntity.ok().body(networkModificationService.getElementaryModification(groupUuid, modificationUuid));
-    }
-
-    @DeleteMapping(value = "/networks/{networkUuid}/modifications")
-    @Operation(summary = "Delete the default modifications group for a network")
-    @ApiResponse(responseCode = "200", description = "Default modifications group deleted for the network")
-    public ResponseEntity<Mono<Void>> deleteModificationGroup(@Parameter(description = "Group UUID") @PathVariable("networkUuid") UUID groupUuid) {
+    @DeleteMapping(value = "/groups/{groupUuid}")
+    @Operation(summary = "Delete the modifications group for a network")
+    @ApiResponse(responseCode = "200", description = "Modifications group deleted for the network")
+    public ResponseEntity<Mono<Void>> deleteModificationGroup(@Parameter(description = "Group UUID") @PathVariable("groupUuid") UUID groupUuid) {
         return ResponseEntity.ok().body(networkModificationService.deleteModificationGroup(groupUuid));
     }
 
-    @GetMapping(value = "/networks/modificationgroups", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/modificationgroups", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
/groups to be consistent with other methods

Reply:
Done